rc

```diff
@@ -31,3 +31,6 @@ if [ ! -e "$codalabdir/venv/bin/register-python-argcomplete" ]; then
 else
     eval "$($codalabdir/venv/bin/register-python-argcomplete cl)"
 fi
+
+# Used the version of `cl` that's installed into `venv` by default.
+. venv/bin/activate
```

Comment:
so if you `. rc` and run `pre-commit.sh`, then you can use the latest version for quick testing.